packages/js/src/ui/context/appearance-context.types.ts

```diff
@@ -0,0 +1,22 @@
+export type CSSProperties = {
```

Comment:
I prefer a single file as well.

Reply:
Yes, I prefer the same to have it in the file where the context is defined, usually because when checking the types I'm also interested really often to see the implementation details.